sources/Application/Utils/HelpLegend.h

```diff
@@ -0,0 +1,115 @@
+#ifndef _HELP_LEGEND_H_
+#define _HELP_LEGEND_H_
+
+#include <string>
+#include <stdlib.h>
+#include <string.h>
+
+class HelpLegend {
+  public:
+	static inline std::string* getHelpLegend(char* fx);
+};
+
+static inline std::string* getHelpLegend(char* fx) {
+	std::string* result = new std::string[3];
+	result[0].assign(fx);
+	result[2].assign("bb at speed aa");
+	if (strcmp(fx, "KILL") == 0) {
```

Comment:
FourCC is an unsigned char in picoTracker. This can be a switch.

Reply:
Sure, the buffer variable argument I'm passing contains the string value matching the effect command, this is why I'm strcmp-ing =)